src/components/Layers/PasteButton.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react'
+import { ToolButton } from '.'
+import { Feature } from 'geojson'
+import { useAppDispatch } from '../../state/hooks'
+import { DiffOutlined } from '@ant-design/icons'
+
+export const PasteButton: React.FC<{ setMessage: (message: string) => void }> = ({ setMessage }) => {
+  const dispatch = useAppDispatch()
+  const [pasteDisabled, setPasteDisabled] = useState(false)
+
+  useEffect(() => {
+    const eventName = 'copy'
```

Comment:
next time you're working @lilitkarapetyan  - I could do with some help here, please.

I'm trying to listen for the clipboard contents changing, so I can enable/disable the paste button.  

But, I can't register for the event properly - my handler never gets called.

I'd be grateful if you could help me to register for the correct event.

Reply:
Aah, it's ok. I've fixed it.  I trigger an event when I write to the clipboard - this invites `Paste` buttons to check the clipboard contents.